### ISSUE TITLE
Release 1.2.3 for live_response_api.py still has code from Release 1.2.1

### ISSUE THREAD

**Issue by darizotas** [carbonblack/carbon-black-cloud-sdk-python#178]
I am still a bit puzzled, but I will try to explain. The file "live_response_api.py" has been modified after Release 1.2.1, more specifically in the commit b15332c, where it changes the references from module endpoint_standard to platform when using the class Device. Shouldn't that have bee included in latter releases? Such as Release 1.2.2 and Release 1.2.3.

I say so, because when I try to to test the jobrunner example, when it tries to establish a live response session it is still querying the deprecated API and comes back with HTTP 401 (Unauthorised). Shouldn't they be updated?

**Comment by avanbrunt-cb** [comment]
@darizotas That change has not been released yet. The Live Response APIs in the CBC SDK version `1.2.x` still require our legacy Live Response key type. In the upcoming `1.3.0` release we are moving to the new v6 Live Response APIs which uses custom API keys which allows us to switch from the legacy device API to the appservices Device API which also uses a custom API key. 

The reason we used the legacy device API was to avoid the need for two API key types during the API workflow. So that was why we had to wait to update the job runner and the Live Response models to the latest platform device model.
